### PR TITLE
Implement modal.onFailure without using jquery

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

File: app/assets/javascripts/blacklight/blacklight.js [hunk 1]
```diff
@@ -327,19 +327,24 @@
...
+          <span aria-hidden="true">&times;</span>
+        </button>
+        </div>
+        <div class="modal-body">
+          <p>Expected a successful response from the server, but got an error</p>
+          <pre>${this.type} ${this.url}\n${jqXHR.status}: ${errorThrown}</pre>
+        </div>`;
+        // const domparser = new DOMParser();
+        // const dom = domparser.parseFromString(contents, "text/html")
+        // const nodes = dom.querySelectorAll('body > *')
```
Comment (dkinzer) [inline app/assets/javascripts/blacklight/blacklight.js:344]: @jcoyne is this left here for documentation purposes?

Comment (jcoyne) [reply app/assets/javascripts/blacklight/blacklight.js:344]: Oops, this is cruft. I'll pull it.

Comment (jcoyne) [reply app/assets/javascripts/blacklight/blacklight.js]: The files in app/assets/javascript are just compiled versions (using `npm run js-compile-bundle`) of what is in `app/javascript`.  In this instance I didn't recompile after changing src.

Comment (jcoyne) [reply app/assets/javascripts/blacklight/blacklight.js]: I recompiled and pushed the results up. That had the effect of removing these old comments.

**Review by dkinzer — APPROVED** [review]

**Pull request merged**